chromedp_test.go

```diff
@@ -1055,3 +1055,18 @@ func TestRunResponse_noResponse(t *testing.T) {
 		}
 	}
 }
+
+func TestCancelBlock(t *testing.T) {
+	alloCtx, cancel := NewExecAllocator(context.Background(), DefaultExecAllocatorOptions[:]...)
+	defer cancel()
```

Comment:
do you need this? you could just do NewContext(context.Background()) I think

Reply:
done, thanks!